public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx

```diff
@@ -127,6 +134,12 @@ export default function KeyValuesTable(props: KeyValuesTableProps) {
                   </LinkValue>
                 </div>
               );
+            } else if (row.type === FlameGraphKey) {
+              valueMarkup = (
+                <div className={styles.flameGraph}>
+                  <FlameGraph data={row.value} getTheme={() => config.theme2} showFlameGraphOnly={true} />
```

Comment:
Why is this hidden in the Accorian values? I mean to me it would seem like a separate `profile` section would make more sense, otherwise there is nothing in the details that would suggest there is some profile until you open the span details.

Reply:
It was the same approach as Pyroscope but have moved it out. It will also have its own section with the new span details.